tests/integration/Skiosa.postman_collection.json

```diff
@@ -1800,6 +1800,228 @@
+							],
+							"path": [
+								"graphql"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "GET - Subscriptions of current User",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status Code is 200\", function() {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"var data = JSON.parse(responseBody);",
+									"",
+									"pm.test(\"Correct Data\", function () {",
+									"    pm.expect(data.data.subscriptions.filter(x => x.id === 1).length).equal(1)",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "graphql",
+							"graphql": {
+								"query": "query subscription {\n    subscriptions {\n        id,\n        name\n    }\n}",
+								"variables": ""
+							}
+						},
+						"url": {
+							"raw": "{{api-url}}/graphql",
+							"host": [
+								"{{api-url}}"
+							],
+							"path": [
+								"graphql"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "GET - Articles of all Subscriptions",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status Code is 200\", function() {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"var data = JSON.parse(responseBody);",
+									"",
+									"pm.test(\"Correct Data\", function () {",
+									"    pm.expect(data.data.articlesOfSubscriptions.filter(x => x.id === 1).length).equal(1)",
+									"    pm.expect(data.data.articlesOfSubscriptions.filter(x => x.id === 3).length).equal(1)",
+									"    pm.expect(data.data.articlesOfSubscriptions.filter(x => x.id === 5).length).equal(1)",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "graphql",
+							"graphql": {
+								"query": "query subscription {\n    articlesOfSubscriptions(PaginationArg: {skip:0, take:3}) {\n        id,\n        title,\n        description,\n    }\n}",
+								"variables": ""
+							}
+						},
+						"url": {
+							"raw": "{{api-url}}/graphql",
+							"host": [
+								"{{api-url}}"
+							],
+							"path": [
+								"graphql"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "POST - Unsubscribe User from Feed",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status Code is 200\", function() {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"var data = JSON.parse(responseBody);",
+									"",
+									"pm.test(\"Correct Data\", function () {",
+									"    pm.expect(data.data.ch.changeSubscription).equal(false);",
```

Comment:
data.data.ch Typo???

Reply:
wonder, how that slipped, fixed in newest commit